addon/helpers/previous.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+import getIndex from '../utils/get-index';
+import createNeedleHaystackHelper from '../-private/create-needle-haystack-helper';
+
+let { isPresent } = Ember;
+
+export function previous(array, currentValue) {
+  let currentIndex = getIndex(array, currentValue);
+  return isPresent(currentIndex) ? array[currentIndex - 1] : null;
```

Comment:
Same here, I think we would want to return the same item if it's already the first item in the array